geoh/geoh.py

```diff
@@ -13,7 +13,12 @@ def geohashes(geojson={}, precision=6, start_precision=2):
     if not polygon:
         return []
 
+    precision_init = precision
+    if precision_init >= 9:
+        precision = precision - 1
```

Comment:
Yeah, I can use `-=` syntax here. 

Value for `precision` gets changed depending on the precision level. Since there will be another evaluation later to decide on further steps, I think it's more clear if we have a value about the initial precision, and use the initial precision in that evaluation step.